GameServerLib/Logic/GameObjects/AttackableUnits/AI/Champion.cs

```diff
@@ -241,6 +247,17 @@ public override void update(float diff)
         {
             base.update(diff);
 
+            if (this.isMovementUpdated()) {
```

Comment:
As I've told you before, it would be better not to have spell specific code in unit specific code. But you don't seem to listen to me nor do more complex changes to make that possible. Nevermind.